apps/studio/components/interfaces/Storage/FilesBuckets.tsx

```diff
@@ -85,28 +112,74 @@ export const FilesBuckets = () => {
                     </TableRow>
                   )}
                   {filesBuckets.map((bucket) => (
-                    <TableRow key={bucket.id}>
+                    <TableRow
+                      key={bucket.id}
+                      onClick={() => {
```

Comment:
understandably we cant make table rows an `a` element, but what i like to do for this if we're making the row a link is to also consider if the meta key is pressed down in the onClick event so that we can still mimic the native browser event of opening links in a new tab if the meta key is pressed

i'll add this in no worries! 🙂 🙏

Reply:
Good catch, thanks! I hate being on the receiving end of that as a user (when it’s not set up).